projects/vdk-plugins/vdk-control-api-auth/src/vdk/plugin/control_api_auth/auth_request_values.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2021 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+from enum import Enum
+
+
+class AuthRequestValues(Enum):
+    """
+    An enum used to the common values used in the redirect logic.
```

Comment:
are we extracting also RedirectAuthentication for reusing?

Reply:
Yes, we will. See https://github.com/vmware/versatile-data-kit/pull/815#discussion_r860761342